base/bitarray.jl

```diff
@@ -1291,11 +1291,6 @@ for f in (:div, :mod)
     end
 end
 
-function (&)(B::BitArray, x::Bool)
```

Comment:
Please see the discussion in #17623, specifically https://github.com/JuliaLang/julia/pull/17623#issuecomment-266635187, https://github.com/JuliaLang/julia/pull/17623#issuecomment-268695364, and https://github.com/JuliaLang/julia/pull/17623#issuecomment-268703295. Best!